spring-cloud-config-server/src/test/java/org/springframework/cloud/config/server/environment/JGitConfigServerTestData.java

```diff
@@ -102,8 +113,11 @@ public static JGitConfigServerTestData prepareClonedGitRepository(Class... sourc
                 .call();
 
         //setup our test spring application pointing to the local repo
+        String[] properties = additionalProperties.toArray(new String[0]);
+        properties = ArrayUtils.add(properties, "spring.cloud.config.server.git.uri:" + "file://" + clonedRepoDir.getAbsolutePath());
```

Comment:
Don't use `ArrayUtils`. Clone `additionalProperties` to a new list and add a new property, then convert to array.

Reply:
done